### PR TITLE
chore: Added rough draft of pieces OS page

### PR DESCRIPTION
**Pull request by mason-at-pieces**

### CODE REVIEW

**Comment by jordan-pieces** [comment]
@georgia-at-pieces can you take a pass at the copy here and review for us?

**Comment by jordan-pieces** [comment]
@mason-at-pieces lets do a few things here: 

1. lets remove `pieces os` from **installation and getting started** in the sidebar 
2. then remove `desktop app` & `pieces os` from **apps & plugins**
3. change the name of `install & get started` to Installation
4. change the name of `overview` to getting started

Screenshot 2023-10-24 at 11 36 02 AM

---

Now on the **Overview** page (which should now be named `Getting Started`):

1. Remove everything from `Bring order to the Chaos` to the bottom of the page, and replace it with your Pieces OS page content 

Let me know if you need any clarification here :) but i think this will make more sense and create a very consumable entry point into our documentation that is less confusing

File: docs/installation-getting-started/pieces-os.mdx [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+Pieces OS's capability extends to managing various machine learning models, either stored locally or in the cloud. It ensures optimal performance and updated models for your applications, bolstering the efficacy and reliability of the machine learning aspects of your development projects.
+
+### Secure and Private
+
+Security and privacy are paramount in Pieces OS's operation. By running locally on your machine, it offers a secure environment for data processing and machine learning model management, ensuring the confidentiality and integrity of your data.
+
+## Getting Started with Pieces OS
+
+### Installation Guide
```
Comment (jordan-pieces) [inline docs/installation-getting-started/pieces-os.mdx:72]: if we do keep this section, we can probably just put the OS direct download links in each of these tabs for Pieces OS

**Review by jordan-pieces — CHANGES_REQUESTED** [review]
there are changes in the comments and see my one note in the code :)

**Review by georgia-at-pieces — APPROVED** [review]

**Review by jordan-pieces — APPROVED** [review]
looks great thanks for all of the updates!!

**Pull request merged**